features/other_a/help.feature

```diff
@@ -35,7 +35,8 @@ Feature: Help
   Scenario: Asked to log in if trying to access the first login page as guest
 
     When I go to the first login help page
-    Then I should see "Sorry, you don't have permission"
+    Then I should see "Log in"
```

Comment:
I was going to suggest "Then I should be on the login page" since "Log in" appears on every page, but, oh, the version on every page is "In." but it might be worth changing anyway, given that's the sole difference?